### PR TITLE
Issue/88 sticky button feedback

### PR DESCRIPTION
**Pull request by thauska**

### CODE REVIEW

**Comment by nicolocarpignoli** [comment]
@gciandro13 can you please review (I'm removing myself as reviewer)

File: components/feedback/stickybutton.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        background-color: var(--primary-color);
+        position: fixed;
+        top: 50%;
+        right: 0px;
+        -webkit-transform-origin: 100% 100%;
+        transform-origin: 100% 100%;
+        -webkit-transform: rotate(-90deg);
+        transform: rotate(-90deg);
+      }
+      .sticky-button:hover .sticky-button:focus {
```
Comment (mingir2) [inline components/feedback/stickybutton.js:23]: I don't think the hover is working as intended. Also, I wonder if we could have box shadows such as following to keep things consistent.
```css
.sticky-button {
...
    box-sizing: border-box;
    box-shadow: -10px 10px 0px black;
...
}
.sticky-button:active {
    box-shadow: -5px 5px 0px #000000;
    webkit-transform-origin: 105% 100%;
    transform-origin: 105% 100%;
    -webkit-transform: rotate(-90deg);
    transform: rotate(-90deg);
}
.sticky-button:hover {
    background-color: var(--primary-color-dark);
}
```

**Review by mingir2 — COMMENTED** [review]
Looks good to me, but I'll let @gciandro13 approve this PR

File: components/feedback/stickybutton.js (hunk 1 above)
Comment (gciandro13) [reply components/feedback/stickybutton.js:23]: It's ok also for me ! I don't know it's ok for @fcor

Comment (fcor) [reply components/feedback/stickybutton.js:23]: Looks good to me @gciandro13 only thing would be that hover effect pointed by @mingir2 😃

**Comment by gciandro13** [comment]
@thauska  PR ok! Great job

**Review by gciandro13 — COMMENTED** [review]
Ok ! I implemented the hover effect suggested by @mingir2

**Pull request merged**